### PR TITLE
UI update headless chrome flags to fix 'yarn run test:oss'

### PR DESCRIPTION
**Pull request by xntrik**
Minor update to address a headless chrome flag issue causing `yarn run test:oss` to error. 

As per https://github.com/ember-cli/ember-cli/pull/8774

I'm unsure if this fix impacts our CI run of the ember tests

Issue flagged as https://github.com/hashicorp/vault/issues/8034

### CODE REVIEW

File: ui/testem.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const config = {
         // --no-sandbox is needed when running Chrome inside a container
         process.env.CI ? '--no-sandbox' : null,
         '--headless',
-        '--disable-gpu',
+        // as per https://github.com/ember-cli/ember-cli/pull/8774
```
Comment (Monkeychip) [inline ui/testem.js]: We don't need to comment out the line here.  You can just include a link to the ember-cli pr in the PR description. Go ahead and remove the `'--disable-gpu--'`

Comment (xntrik) [reply ui/testem.js]: Okies, removed that line, and updated the original PR comment with that ember-cli link.

Comment (Monkeychip) [reply ui/testem.js]: Can you also remove the comment (sorry, I didn't make that clear).  With the ember-cli link now in the PR comment, no need to have it in the code.  Thanks!

Comment (xntrik) [reply ui/testem.js]: That's sort of what I thought, but didn't quite get - update coming.

Comment (xntrik) [reply ui/testem.js]: done.

**Pull request merged**